### ISSUE TITLE
More Redux for ranged

### ISSUE THREAD

**Issue by 01der** [darklordabc/Legends-of-Dota-Redux#1526]
I suggest that more abilities get the furyswipe/essenceshift/jingu treatment, 
where ranged has 50% less dmg/dps/chance:

- all bashes: slardar, void, spiritbreaker
- pangolier passives (these also shouldn't proc off illusions imo)
- abbadon passive
- underlord passive

slardar bash is actually broken on ranged, in the sense, that it procs the guaranteed bash for every projectile till the first one hits its target.

**Comment by Wouterz90** [comment]
Agreed. For Slardar, I've tested it, it seems like a 0 second bash.

Not going to rework pangolier passives, so I guess that won't be changed.

**Comment by Wouterz90** [comment]
Lucky shot, time lock and bash of the deep should be replaced soon. Others later.
